var/spack/repos/builtin/packages/llvm-openmp/package.py

```diff
@@ -76,10 +93,31 @@ def patch(self):
             os.rename(cmake_mod_dir, os.path.join(self.stage.path, "cmake"))
 
     def cmake_args(self):
-        cmake_args = []
+
+        # Disable LIBOMP_INSTALL_ALIASES, otherwise the library is installed as
+        # libgomp alias which can conflict with GCC's libgomp.
+        cmake_args = ["-DLIBOMP_INSTALL_ALIASES=OFF"]
```

Comment:
I think this should be moved inside the multicompat handling, or not?
```suggestion
        cmake_args = []
```

Reply:
I made this change:   cmake_args = []